Nodejs/Product/Nodejs/Project/NodeModulesNode.cs

```diff
@@ -148,7 +149,7 @@ private void NpmController_FinishedRefresh(object sender, EventArgs e)
             ReloadHierarchySafe();
         }
 
-        public INpmController NpmController => this._npmController;
+        public INpmController NpmController { get; }
```

Comment:
Personally, I find it helpful to extract this sort of semantic-preserving refactoring into one or more separate commits.